templates/konvoy-ui.yaml

```diff
@@ -27,6 +27,8 @@ spec:
       createObservableClusterCRD: false
       mode: konvoy
       logoutRedirectPath: /ops/landing
+      clusterPollingInterval: 3000
```

Comment:
Since this is the default 
```
clusterPollingInterval: 3000
clientPollingInterval: 3000
```
in the [chart](https://github.com/mesosphere/charts/blob/bb2cbb5618810e5cb8b21303c072c69c7799183c/stable/kommander/values.yaml#L7-L8) this change is not needed.

Reply:
+1 on @joejulian comment